### PR TITLE
Add French translation

### PR DESCRIPTION
**Pull request by Feinte75**
This PR includes a good starting point for a French translation

### CODE REVIEW

**Comment by PKEv** [comment]
it's better translate here: https://hosted.weblate.org/projects/qtodotxt/translations/fr/

File: i18n/fr_FR.ts [hunk 1]
```diff
@@ -5,51 +5,51 @@
     <message>
         <location filename="main_controller.py" line="70"/>
         <source>Show &amp;Filters</source>
-        <translation type="unfinished"></translation>
+        <translation>Montrer les filtres</translation>
```
Comment (PKEv) [inline i18n/fr_FR.ts]: It is necessary to leave a symbol "&amp;", it indicates what character is used for quick access

**Comment by Feinte75** [comment]
Thanks for the feedback, I'll translate on the website as it appears to be simpler :).

**Comment by PKEv** [comment]
Thank you for supporting the project, your work is already in the current version. 
you can evaluate the result obtained by using the instructions on this link: https://github.com/QTodoTxt/QTodoTxt#running-from-sources

**Comment by Feinte75** [comment]
I tested and had some problems :
* I ran QTodoTxt from source successfully but the language defaulted to English
* I realized that the translation file "fr_FR.ts" was not compiled so i used the script "pylupdate.py" with upd parameter, it completely broke the translation file adding "obsolete" and "unfinished" tags to 90% of the translations messages and even removing some translations.
* I reverted to the complete translation file and used pylupdate with fix parameter. It generated the ".qm" file, I then launched the app and had only some translations applied. 

Maybe it has to do with line numbers in the translation file not pointing to the correct strings ?

**Comment by PKEv** [comment]
You're right, I'll try to find out

**Comment by PKEv** [comment]
Yes, you're right, I forgot that we must also compile it.
A big difference is due to the fact that the last time the file transfer to the French was formed a long time ago, and there were new lines

**Comment by PKEv** [comment]
Please actualize translation on site  https://hosted.weblate.org/projects/qtodotxt/translations/fr/

**Comment by Feinte75** [comment]
Rough traduction done !

**Pull request closed without merging**